game-core/src/main/java/games/strategy/triplea/attachments/AbstractConditionsAttachment.java

```diff
@@ -289,13 +279,6 @@ public int getChanceIncrementOnFailure() {
     return m_chanceIncrementOnFailure;
   }
 
-
-  @GameProperty(xmlProperty = true, gameProperty = true, adds = false)
-  private void setChanceDecrementOnSuccess(final String value) {
-    setChanceDecrementOnSuccess(getInt(value));
-  }
```

Comment:
This method was identified by static analysis as unused (presumably the annotation was preventing the static analyzer from identifying that before this change).  I think it became unused when the `MutableProperty#ofMapper()` helper was added.